redfish-core/lib/log_services.hpp

```diff
@@ -6970,6 +7060,7 @@ inline void fillSystemHardwareIsolationLogEntry(
         "/redfish/v1/Systems/system/LogServices/HardwareIsolation/Entries/{}",
         dbusObjIt->first.filename());
     entryJson["Id"] = dbusObjIt->first.filename();
+    entryJson["MessageId"] = "OpenBMC.0.6.GardRecord";
```

Comment:
On error to fill in the message are we going to end up with an entry in the Members array with a MessageId, etc. but no actual Message or MessageArgs?

Reply:
Message will be the one in openbmc.ison file. There will be 2 args in MessageArgs. But they will be empty

Reply:
If it failed due to loading the message from openbmc.json, yes it will not have message and messageargs

Reply:
So should i move loading the messageid to the same function where we update the message and messageargs. So it is consistent?

Reply:
I would vote for doing the message handling similar to the other messages.

Reply:
I see other caller of fillMessageArgs() that waits to fill in all of the fields until after any errors could have happened. 

https://github.com/ibm-openbmc/bmcweb/blob/d2fe0d8eb1940375b34a0b6d5d6a5a93097d0910/redfish-core/lib/log_services.hpp#L5398

Reply:
Anyway as we are raising an internal error, believe not able to wait for it to load should not be an issue.